### DISCUSSION TITLE
View Recipe from meal plan

### DISCUSSION THREAD

**Discussion by labsnoir** [TandoorRecipes/recipes#1385] [Q&A] [answered]
Is it possible to view the recipe from the calendar plan? Perhaps I am missing something, I always need to go back to "Cookbook" and search for the recipe there.

Nevertheless, TandoorRecipes is a really great tool - keep on with the good work :)

**Comment by vabene1111** [comment, marked as answer]
there is a bug #1093, the next release will have an intermediate fix but we are working on it

**Reply by labsnoir** [reply]
Thank you for the quick answer :)
